apps/authentication/index.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import apolloMiddleware from 'react/apollo/middleware';
+
+import { StaticRoutes } from './client/index';
+
+const middlewareStack = [
+  apolloMiddleware,
+];
+
+const app = express();
+
+app.set('views', `${__dirname}/templates`);
+app.set('view engine', 'jade');
+
+app
+  .get(/^\/(sign_up|log_in|forgot)/, ...middlewareStack, (req, res) => req.apollo.render(StaticRoutes).then(apollo => res.render('index', { apollo })));
```

Comment:
interesting location for a line break :) some more might make this a bit more legible.